source/view.js

```diff
@@ -1245,6 +1245,7 @@ view.ModelFactoryService = class {
         this.register('./npz', [ '.npz', '.h5', '.hd5', '.hdf5' ]);
         this.register('./dl4j', [ '.zip' ]);
         this.register('./mlnet', [ '.zip' ]);
+        this.register('./mslite', [ '.ms']);
```

Comment:
This is confusing. `.ms` is already used as an extension by other formats. The FlatBuffer has no `file_identifier` which makes it impossible to distinguish these formats. 

- [ ] can the file extension be changed to `.mslite` to be consistent with the module name?
- [ ] can a `file_identifier` be added to `schema.fbs`?

Reply:
In model.fbs, the “version” attribute of table MetaGraph can be used to determine whether a ’.ms' file is a MindSpore Lite model.

Also, in the line 38 of mslite.js, I've supplement a format matching judgement for the ".ms" file.

Reply:
This does not work as you first need to successfully parse the FlatBuffers file to check the version. The file might not be a valid FlatBuffers file or in a different format that does not match the schema. The correct way to do this is using `file_identifier` and `file_extension`. Existing files might have to get recreated (great time to also fix the file extension from `.ms` to `.mslite`).

Reply:
Since such changes will affect the compatibility of many current MindSpore Lite models, this PR needs to be suspended temporarily.